index.bs

```diff
@@ -281,6 +281,33 @@ In these cases, features should be designed such that it is possible for
 code to gracefully fail or be polyfilled.
 See [[#feature-detect]].
 
+<h3 id=new-features>New features can add functionality to the Web</h3>
+
+A new Web feature can add capabilities to the Web with little concern about the effect on existing content.
+The Web includes many extension points that allow for additions.
+This includes
+HTML elements and attributes;
+DOM interfaces and functions; and,
+CSS rules and selectors.
```

Comment:
```suggestion
This includes:
- New HTML elements, attributes, attribute values
- New CSS rules, pseudo-classes, properties
- New DOM interfaces and functions
- New DOM events
```

Reply:
or replace the list with a reference to https://html.spec.whatwg.org/multipage/introduction.html#extensibility